.github/workflows/ping.yml

```
@@ -0,0 +1,11 @@
+name: Ping
+on:
+  schedule:
+    - cron: '0/10 * * * *'
+jobs:
+  build:
+    name: ping heroku server
+    runs-on: ubuntu-latest  
+    steps:
+      - name: cURL request
+        run: curl -X GET https://mftp.herokuapp.com/
```

```suggestion
        run: curl mftp.herokuapp.com -o NUL
```

Do you think we should use the original command, as it is from the script?